content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,206 @@
+
+## Container & Image scanning
+
+Scanning a container image can prevent critical vulnerabilities from being
+deployed to the cluster alongside the container image. Image scanning should be
+completed before deploying a container image to a cluster and is usually done
+as part of the deployment process in a CI/CD pipeline. The purpose of an image
+scan is to obtain information about possible vulnerabilities and their
+prevention in the container image, such as a
+[Common Vulnerability Scoring System (CVSS)](https://en.wikipedia.org/wiki/Common_Vulnerability_Scoring_System)
+score. If the result of the image scans is combined with the pipeline
+compliance rules, only properly patched container images will end up in
+Production.
+
+The blog post
+[A Closer Look at NSA/CISA Kubernetes Hardening Guidance](/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/#building-secure-container-images)
+## Admission Controllers
+
+The following are a number of admission controllers that could be considered to
+enhance the security posture of your application. This list includes
+controllers that may be referenced in other parts of this document:
+
+- [`AlwaysPullImages`](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages)
+  \- Enforces the latest version of a tagged image is used, and the deployer has
+    permissions to use the image.
+- [`CertificateApproval`](/docs/reference/access-authn-authz/admission-controllers/#certificateapproval)
+  \- Performs additional authorization checks to ensure the approving user has
+    permission to approve certificate request.
+- [`CertificateSigning`](/docs/reference/access-authn-authz/admission-controllers/#certificatesigning)
+  \- Performs an additional authorization checks to ensure the signing user has
+    permission to sign certificate requests.
+- [`CertificateSubjectRestriction`](/docs/reference/access-authn-authz/admission-controllers/#certificatesubjectrestriction)
+  \- Rejects any certificate request that specifies a 'group' (or 'organization attribute')
+    of `system:masters`.
+- [`DenyServiceExternalIPs`](/docs/reference/access-authn-authz/admission-controllers/#denyserviceexternalips)
+  \- Prevents usage of `externalIPs` services
+- [`EventRateLimit`](/docs/reference/access-authn-authz/admission-controllers/#eventratelimit)
+  \- Rate limts events to the API server.
+- [`ImagePolicyWebhook`](/docs/reference/access-authn-authz/admission-controllers/#imagepolicywebhook)
+  \- Allows enforcing additional controls for images through webhooks.
+- [`MutatingAdmissionWebhook`](/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook)
+  \- Allows the use of custom controllers through webhooks, these controllers
+    may mutate requests that it reviews.
+- [`NodeRestriction`](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)
+  \- Restricts Kubelet permissions based of deployments upon their nodes.
```

Comment:
```suggestion
\- This admission controller can increase security when labels with a `node-restriction.kubernetes.io/` prefix are used. Only the administrator can update those labels and even the kubelet can not modify them. It can prevent an attacker with access to kubelet credentials from modifying node labels and thus influence the pod placement, when e.g. `NodeSelectors` are used.
```

Reply:
Resolve by https://github.com/kubernetes/website/pull/33992/commits/2672d212fa9585eea7acda2028c2757461ecd2f7.